addon/.eslintrc.cjs

```diff
@@ -2,29 +2,76 @@
 
 module.exports = {
   root: true,
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    ecmaVersion: 'latest',
-  },
-  plugins: ['ember'],
-  extends: [
-    'eslint:recommended',
-    'plugin:ember/recommended',
-    'plugin:prettier/recommended',
-  ],
-  env: {
-    browser: true,
-  },
-  rules: {},
+  // Only use overrides
+  // https://github.com/ember-cli/eslint-plugin-ember?tab=readme-ov-file#gtsgjs
   overrides: [
+    {
+      files: ['**/*.js', '**/*.ts'],
```

Comment:
@elwayman02 I think this should be
```suggestion
      files: ['src/**/*.js', 'src/**/*.ts'],
```
to fix that pesky `Parsing error: 'import' and 'export' may appear only with 'sourceType: module'`

I'll check later if it's issue in blueprint and submit PR upstream

Reply:
Looks like the issue was the node file config not being setup to handle `.mjs` files...turns out if I remove the rollup file from that config for now, everything passes. There's a more correct answer (fixing the config to parse mjs properly), but this unblocks me and the blueprint will need that fix anyway so I'll pick it up eventually here.

Reply:
@elwayman02 ahha, ok, interesting. We need to fix this up separately in the blueprint. Thanks for digging in!